ddtrace/internal/_rand.pyx

```diff
@@ -55,6 +55,7 @@ import os
 
 from ddtrace import compat
 
+
```

Comment:
formatter?

Reply:
I think so 🤔 definitely didn't do it by hand